src/main/java/com/flightstats/hub/webhook/WebhookLeaderState.java

```diff
@@ -28,13 +31,40 @@ public ActiveWebhooks(
         this.localHostProperties = localHostProperties;
     }
 
-    boolean isActiveWebhook(String webhookName) {
+    public RunningState getState(String webhookName) {
+        return RunningState.builder()
+                .leadershipAcquired(hasLeader(webhookName))
+                .runningServers(getServers(webhookName))
+                .build();
+    }
+
+    private boolean hasLeader(String webhookName) {
         return webhookLeaderLocks.getWebhooks().contains(webhookName);
     }
 
-    public Set<String> getServers(String name) {
+    private Set<String> getServers(String name) {
         return webhookLeaderLocks.getServerLeases(name).stream()
                 .map(server -> server + ":" + localHostProperties.getPort())
                 .collect(toSet());
     }
+
+    @Builder
+    @Wither
+    @Value
+    public static class RunningState {
+        boolean leadershipAcquired;
+        Set<String> runningServers;
+
+        public boolean isRunningOnSingleServer() {
+            return isLeadershipAcquired() && getRunningServers().size() == 1;
+        }
+
+        public boolean isStopped() {
+            return !isLeadershipAcquired() && getRunningServers().isEmpty();
```

Comment:
Depending on the construction, runningServers could be null. You could add `@Builder.Default` to the instance declaration.
```
@Builder.Default
Set<String> runningServers() = Collections.emptySet();
```